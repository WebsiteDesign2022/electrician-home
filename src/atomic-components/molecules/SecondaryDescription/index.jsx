import React from "react"

const SecondaryDescription = ({ text }) => (
<p align="left" className="text-black">
    {text}
</p>
);

export default SecondaryDescription;
