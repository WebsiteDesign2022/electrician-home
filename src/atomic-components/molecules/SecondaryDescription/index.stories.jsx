import React from 'react';

import SecondaryDescription from '.';

export default {
  title: 'Electrician-Home/molecules/SecondaryDescription',
  component: SecondaryDescription,
  parameters: {
    // More on Story layout: https://storybook.js.org/docs/react/configure/story-layout
    layout: 'fullscreen',
  },
};

const Template = (args) => <SecondaryDescription {...args} />;

export const Primary = Template.bind({});
// More on args: https://storybook.js.org/docs/react/writing-stories/args
Primary.args = {
    text: 'Primary'
};
