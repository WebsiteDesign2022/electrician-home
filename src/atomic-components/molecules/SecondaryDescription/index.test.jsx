import TestRenderer from 'react-test-renderer';

import SecondaryDescription from '.';

describe("SecondaryDescription", () => {
    test("renders SecondaryDescription without text and matches snapshot", () => {
        const secondaryDescription = TestRenderer.create(
            <SecondaryDescription/>
        );

        expect(secondaryDescription.toJSON()).toMatchSnapshot();
    });

    test("renders SecondaryDescription with text and matches snapshot", () => {
        const secondaryDescription = TestRenderer.create(
            <SecondaryDescription text="secondary-description-text"/>
        );

        expect(secondaryDescription.toJSON()).toMatchSnapshot();
    });
});
