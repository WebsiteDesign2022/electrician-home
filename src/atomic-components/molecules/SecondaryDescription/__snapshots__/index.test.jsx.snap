// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SecondaryDescription renders SecondaryDescription with text and matches snapshot 1`] = `
<p
  align="left"
  className="text-black"
>
  secondary-description-text
</p>
`;

exports[`SecondaryDescription renders SecondaryDescription without text and matches snapshot 1`] = `
<p
  align="left"
  className="text-black"
/>
`;
